pkgs/development/python-modules/seahub/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, lib, fetchFromGitHub, python3Packages, makeWrapper }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "seahub";
+  version = "8.0.4";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seahub";
+    rev = "v${version}-server";
+    sha256 = "0wi1m6k1km2n17s19jxg05jhmv7x59w92mijnay786ghhakd9y1r";
+  };
+
+  dontBuild = true;
+  doCheck = false;
```

Comment:
Why did you disable tests? Please add a comment with that information.

Reply:
Unfortunately tests requires a fully setup ccnet. Comment added.